packages/ui/cypress/support/page-objects/multisigPage.ts

```diff
@@ -8,5 +9,13 @@ export const multisigPage = {
   transactionList: () => cy.get('[data-cy=container-transaction-list]', { timeout: 20000 }),
   pendingTransactionItem: () => cy.get('[data-cy=container-pending-tx-item]'),
   reviewButton: () => cy.get('[data-cy=button-review-tx]'),
-  setIdentityMenuOption: () => cy.get('[data-cy=menu-option-set-identity]')
+  setIdentityMenuOption: () => cy.get('[data-cy=menu-option-set-identity]'),
+
+  // multisig details elements
+  multisigDetailsContainer: () => cy.get('[data-cy=container-multisig-details]'),
+  multisigAccountSummary: () => cy.get('[data-cy=container-multisig-account-summary]'),
+  thresholdListItem: () => cy.get('[data-cy=list-item-threshold]'),
+  balanceListItem: () => cy.get('[data-cy=list-item-balance]'),
+  signatoriesAccordion: () => cy.get('[data-cy=accordion-signatories]'),
+  expandSignatoriesIcon: () => cy.get('[data-cy=icon-expand-signatories-summary]')
```

Comment:
I'm not utilizing all of the above element locators yet but we can soon on other test scenarios, thought it made sense to add them all whilst I was in that area of code